### PR TITLE
docs: adds tech on readme file

### PR DESCRIPTION
**Pull request by paschalidi**
adds read me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+* react
+* typescript
+* cra
+* lerna 
+* we only use hooks
+* styled-components
+* Firebase
+* storybook
+* jest for unit testing 
+* e2e with cypress
```
Comment (paschalidi) [inline README.md:16]:
```suggestion
* `react`
* `typescript`
* cra
* `lerna`
* we only use hooks
* `styled-components`
* `firebase`
* `storybook`
* `jest` for unit testing 
* e2e with `cypress`
```

You could mark the libraries as above

**Review by paschalidi — COMMENTED** [review]
Comments to myself

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+
+still to choose: 
+* state-managemenrs
```
Comment (AdrianValdes) [inline README.md:19]: There is a typo here

**Review by AdrianValdes — CHANGES_REQUESTED** [review]
Thanks for adding the techs on the README. I found a little typo, you may want to change it.

File: README.md (hunk 2 above)
Comment (paschalidi) [reply README.md]: @AdrianValdes thanks

Comment (paschalidi) [reply README.md]: @AdrianValdes thanks for the comment

**Comment by paschalidi** [comment]
@AdrianValdes I looked in your comments

**Review by AdrianValdes — APPROVED** [review]
It looks very good. Thanks.

**Pull request merged**